models/standalone-price/src/types.ts

```diff
@@ -4,13 +4,18 @@ import {
   CustomerGroup,
   Channel,
 } from '@commercetools/platform-sdk';
+import { TChannel } from '@commercetools-test-data/channel';
 import {
   TMoneyGraphql,
   TReferenceGraphql,
+  TReferenceRest,
 } from '@commercetools-test-data/commons';
 import type { TBuilder } from '@commercetools-test-data/core';
+import { TCustomerGroup } from '@commercetools-test-data/customer-group';
 
 // Base representation
+// TODO: This doesn't seem to be right. I don't understand why the default type should have customized fields.
+// It seems to me that's something only transformed versions of the model should have.
 export type TStandalonePrice = Omit<
```

Comment:
This is something I don't understand. Maybe it was set up like this to notice these properties have a different type in the alternative representations of the model (rest and graphql) but this pattern is actually missleading.